routes/web.php

```diff
@@ -27,6 +28,10 @@
 	->only(['index', 'store', 'edit', 'update', 'destroy'])
 	->middleware(['auth', 'verified']);
 
+Route::resource('comments', CommentController::class)
+    ->only(['create', 'store', 'edit', 'update', 'destroy'])
```

Comment:
you only need store, update, and destroy, we won't have a specific page to show a form for creating / editing comments., it will be done in the chirps view